package/upgrade/upgrade_node.sh

```diff
@@ -605,6 +615,8 @@ command_single_node_upgrade() {
 
   convert_nodenetwork_to_vlanconfig
 
+  # update max-pods to 200 #
+  set_max_pods
```

Comment:
Should we move this before `upgrade_rke2` function?
Putting the call here then the change takes effect after rebooting. In theory there is no difference.

Reply:
sounds good. makes it better to organize the rke2 changes together.